var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -17,6 +17,8 @@ class Charliecloud(AutotoolsPackage):
     tags = ["e4s"]
 
     version("master", branch="master")
+    version("0.29", sha256="c89562e9dce4c10027434ad52eaca2140e2ba8667aa1ec9eadf789b4d7c1a6db")
```

Comment:
One thing to note is that 0.29 introduces a git build cache that requires git 2.28.1 or greater.

We should add something like:

```
depends_on("git@2.28.1:", type="run", when="@0.29:")
```

Reply:
done.

Reply:
It doesn't need to be 2.28.1 exactly, and in fact if there's a choice, the newest available Git is likely the best one.